src/ml-search.service.js

```diff
@@ -629,9 +629,9 @@
      */
     getCombinedQuery: function getCombinedQuery(includeOptions) {
       var d = $q.defer(),
-          combined = {
-            search: { query: this.getQuery() }
-          };
+        combined = {
+          search: { query: this.getQuery() }
+        };
 
```

Comment:
I don't know, what version of jshint are you using? Does it respect the project config?

Here's what I have, with no complaints:

    $ jshint --version
    jshint v2.6.3
    $ jshint src/ml-search.service.js 

    $ npm ls | grep jshint
    ├─┬ gulp-jshint@1.11.0
    │ ├─┬ jshint@2.8.0
    $ gulp jshint
    [15:59:06] Using gulpfile ~/Projects/ml-search-ng/gulpfile.js
    [15:59:06] Starting 'jshint'...
    [15:59:06] Finished 'jshint' after 7.91 ms